flow/visualize/transfer/util.py

```diff
@@ -6,13 +6,13 @@
     VEH_PER_HOUR_BASE_27414345, VEH_PER_HOUR_BASE_27414342
 
 
-def make_inflows(penetration_rate=0.1, flow_rate_coef=1.0, departSpeed=20, on_ramp=False):
+def make_inflows(pr=0.1, fr_coef=1.0, departSpeed=20, on_ramp=False, apr=0.0):
```

Comment:
@kjang96 made some changes to how we setup the inflows for transfer, but i think everything should be sensible